fuzzers/fuzzbench_qemu/src/fuzzer.rs

```diff
@@ -255,7 +255,7 @@ fn fuzz(
 
     let map_feedback = MaxMapFeedback::new_tracking(&edges_observer, true, false);
 
-    let calibration = CalibrationStage::new(&map_feedback);
+    let calibration = CalibrationStage::new(&map_feedback, true);
```

Comment:
We could just have `::new` and `::ignore_stability` or seomthing, booleans in the API isn't a good choice